MetaMask/metamask-mobile
# fix: 1078 mixpanel delete data

**Description**
This PR changes the Mixpanel endpoint for deleting user data (MetaMetrics) to a proxy endpoint.

**Screenshots/Recordings**
## Successful delete data request UI
![Simulator Screen Shot - iPhone 14 - 2023-09-01 at 17 32 59](https://github.com/MetaMask/metamask-mobile/assets/141057783/ed791f96-6ecb-4913-a54f-f07f0a005823)


## RECORDING

https://github.com/MetaMask/metamask-mobile/assets/141057783/5bb71f39-f3bd-4858-8cde-84fafc886b1f



**Issue**

fixes [1078](https://github.com/MetaMask/mobile-planning/issues/1078)

**Testing**
1) Go to settings screen > Security & Privacy
2) Click on "Delete MetaMetrics Data" Button
3) Confirm on Popup
4) (Optional) Check Network tab in Flipper to watch POST & GET to proxy endpoint
5) Should see UI update with requested delete data date as in screenshot

**Checklist**
